.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
There is, I can look into mimicking those settings in vscode configuration file, maybe under a separate issue. I know you prefer not to deal with code ;)